openshift/templates/backup/backup-deploy.yaml

```diff
@@ -0,0 +1,432 @@
+  - name: CONFIG_MOUNT_PATH
+    displayName: Config Mount Path
+    description: The path to use to mount the config file.
+    required: true
+    value: "/"
+  - name: BACKUP_VOLUME_NAME
+    displayName: Backup Volume Name
+    description:
+      The name of the persistent volume used to store the backups.  Please
+      note, when using the recommended nfs-backup storage class the name of the pvc
+      MUST be taken from the manually provisioned claim; nfs-backup storage MUST be
+      provisioned manually.
+    required: true
+    value: bk-jcxjin-dev-r9c53stkggh6
+  - name: BACKUP_VOLUME_SIZE
+    displayName: Backup Volume Size
+    description:
+      The size of the persistent volume used to store the backups, e.g. 512Mi,
+      1Gi, 2Gi.  Ensure this is sized correctly.  Refer to the container documentation
+      for details.
+    required: true
+    value: 5Gi
+  - name: BACKUP_VOLUME_CLASS
+    displayName: Backup Volume Class
+    description:
+      The class of the persistent volume used to store the backups; netapp-block-standard,
+      netapp-file-standard, nfs-backup.  Please note, nfs-backup storage is the recommended
+      storage type for backups.  It MUST be provisioned manually through the OCP catalog
+      via the 'BC Gov NFS Storage' template.  nfs-backup storage CANNOT be automatically
+      provisioned by this template.
+    required: true
+    value: nfs-backup
+  - name: VERIFICATION_VOLUME_NAME
+    displayName: Verification Volume Name
+    description:
+      The name for the verification volume, used for restoring and verifying
+      backups.  When using the recommend nfs-backup storage class for backups, this
+      volume MUST be either netapp-file-standard or netapp-block-standard storage; netapp-block-standard
+      is recommended (it has far better performance).
+    required: false
+    value: backup-verification
+  - name: VERIFICATION_VOLUME_SIZE
+    displayName: Backup Volume Size
+    description:
+      The size of the persistent volume used for restoring and verifying
+      backups, e.g. 512Mi, 1Gi, 2Gi.  Ensure this is sized correctly.  It should be
+      large enough to contain your largest database.
+    required: true
+    value: 1Gi
+  - name: VERIFICATION_VOLUME_CLASS
+    displayName: Backup Volume Class
+    description:
+      The class of the persistent volume used for restoring and verifying
+      backups; should be one of netapp-block-standard or netapp-file-standard.  netapp-block-standard
+      performs better.
+    required: true
+    value: netapp-file-standard
+  - name: VERIFICATION_VOLUME_MOUNT_PATH
+    displayName: Verification Volume Mount Path
+    description:
+      The path on which to mount the verification volume.  This is used by
+      the database server to contain the database configuration and data files. For
+      Mongo, please use /var/lib/mongodb/data . For MSSQL, please use /var/opt/mssql/data
+    required: true
+    value: "/var/lib/pgsql/data"
```

Comment:
Shouldn't we use `/var/opt/mssql/data` here as our default?

Reply:
yep, I'll fix this.